src/foam/u2/dialog/InlineNotificationMessage.js

```diff
@@ -124,10 +131,15 @@ foam.CLASS({
                 .attrs({ src: this.iconImage$ })
               .end()
             .end()
-            .start()
+            .startContext({ controllerMode: this.ControllerMode.VIEW })
               .addClass(this.myClass('message'))
-              .add(this.message$)
-            .end()
+              .callIf(this.customView, function() {
```

Comment:
This can just be a `callIfElse()`. Also I don't think we should have 2 different properties. We could just check for the type of the prop with the view/string and then render it appropriately